### DISCUSSION TITLE
MacOS Quarkus & Embedded MongoDB Issue

### DISCUSSION THREAD

**Discussion by v-shalimov** [flapdoodle-oss/de.flapdoodle.embed.mongo#396] [Q&A] [answered]
Hi All!

I am developing using Quarkus on MacBook Pro. When I used Quarkus version 2.7.1-Final with embedded MongoDB (de.flapdoodle.embed.mongo) version 3.2.0 included - everything was OK. But when I upgrade it to the latest version (2.8.1-Final) with embedded MongoDB version 3.3.0 I am constantly getting an error:
```
Extract /Users/vshalimov/.embedmongo/tools/db/mongodb-database-tools-macos-x86_64-100.5.1.zip START
2022-04-22 13:00:03,524 WARN  [de.fla.emb.pro.ext.AbstractExtractor] (main) 
--------------------------
If you get this exception more than once, you should check if the file is corrupt.
If you remove the file (/Users/vshalimov/.embedmongo/tools/db/mongodb-database-tools-macos-x86_64-100.5.1.zip), it will be downloaded again.
--------------------------: java.io.IOException: Input is not in the .gz format
```
Will be very thankful for any assistance in that regard.

**Comment by michaelmosmann** [comment, marked as answer]
related issue https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo/issues/397

**Comment by michaelmosmann** [comment]
reopen if needed..
